src/main/java/com/agilethought/atsceapi/service/UserServiceImpl.java

```diff
@@ -1,5 +1,74 @@
 package com.agilethought.atsceapi.service;
 
-public class UserServiceImpl {
+import com.agilethought.atsceapi.exceptions.BadRequestException;
+import com.agilethought.atsceapi.exceptions.NotFoundException;
+import com.agilethought.atsceapi.model.LoginData;
+import com.agilethought.atsceapi.model.User;
+import com.agilethought.atsceapi.repository.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+
+import java.util.List;
+

```

Comment:
There's way too many empty spaces in all your classes, please go through EVERY SINGLE ONE of your classes and interfaces and hit "Ctrl+Shift+F." This will properly format your code.